monitoring/accessing-third-party-uis.adoc

```diff
@@ -21,12 +21,7 @@ Default access to the third-party monitoring interfaces might be removed in futu
 
 [NOTE]
 ====
-The Grafana instance that is provided with the {product-title} monitoring stack, along with its dashboards, is read-only.
-====
-
-[NOTE]
-====
-The Grafana dashboard includes Kubernetes and `cluster-monitoring` metrics only. Additional platform components are included in *Monitoring* → *Dashboards* in the {product-title} web console.
+The Grafana instance that is provided with the {product-title} monitoring stack, along with its dashboards, is read-only. The Grafana dashboard includes Kubernetes and `cluster-monitoring` metrics only.
```

Comment:
I combined these two notes so that we don't have back-to-back notes here. We try to avoid that in the docs if possible.

Reply:
Suppose it would be better move 'Dashboards for some additional platform components are included in *Monitoring* → *Dashboards* in the {product-title} web console.' from line 15 to NOTE on line 24 as before